pkg/netpol/common/netpol_testing_common.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// GenerateActualOutputFile generates actual output file
+func GenerateActualOutputFile(t *testing.T, testName, output, filePath string) {
+	fp, err := os.Create(filePath)
+	require.Nil(t, err, "test %q: error creating file %q", testName, filePath)
+	_, err = fp.WriteString(output)
+	require.Nil(t, err, "test %q: error writing output to file %q", testName, filePath)
+}
```

Comment:
I think we can have a more general `func` for just writing to file.
No need for the `require` statements here, the test should not fail due to issue in writing to file.
It can return an `error` value, and in the tests you can print to log that there was issue in writing to file, if `err` isn't `nil`.
This function does not have to be specific for tests.